### PR TITLE
Starting to add some 2024 info

### PR DESCRIPTION
**Pull request by wolframkriesing**

### CODE REVIEW

File: src/lib/participants/registration.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-export const registrationOpensAt = +new Date('2023-05-01 00:00:00Z');
-export const eventStartsAt = +new Date('2023-06-30 09:30:00Z');
-export const registrationClosessAt = +new Date('2023-06-26 17:00:00Z');
+export const registrationOpensAt = +new Date('2024-05-01 00:00:00Z');
+export const eventStartsAt = +new Date('2024-06-30 09:30:00Z');
```
Comment (Narigo) [inline src/lib/participants/registration.ts:2]: We don't know this yet 😅

File: src/lib/participants/registration.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+export const registrationClosessAt = +new Date('2023-04-26 17:00:00Z');
```
Comment (Narigo) [inline src/lib/participants/registration.ts:3]: That's... interesting...! I don't know whether I tested the case of having registration close before it opens... 😬

**Review by Narigo — APPROVED** [review]
Thanks a lot for picking this up! I like the changes 👍

File: src/lib/participants/registration.ts (hunk 1 above)
Comment (wolframkriesing) [reply src/lib/participants/registration.ts:2]: its in the future, that is good enough for now

File: src/lib/participants/registration.ts (hunk 2 above)
Comment (wolframkriesing) [reply src/lib/participants/registration.ts:3]: not relevant yet... we should change it once we know

**Pull request merged**